libs/ui/src/lib/tabs/cvi-tabs.component.html

```diff
@@ -0,0 +1,74 @@
+<div class="cvi-tabs__list" role="tablist" *ngIf="currentWindowWidth >= 992">
+  <button
+    *ngFor="let tab of allTabs; let i = index"
+    dataAttribute="tab-button-{{ i }}"
+    [ngClass]="{ active: isTabSelected(i), underlined: isTabSelected(i) }"
+    (click)="makeActive(i)"
+    type="button"
+    #tabButton
+    id="{{ baseId }}-tab-{{ i }}"
+    [attr.aria-selected]="isTabSelected(i)"
+    role="tab"
+    [attr.tabindex]="isTabSelected(i) ? null : '-1'"
+    (keyup.arrowleft)="makeActivePrev($event)"
+    (keyup.arrowright)="makeActiveNext($event)"
+    [attr.aria-controls]="baseId + '-tabpanel-' + i"
+    class="cvi-tabs__tab-btn"
+  >
+    {{ tab.title }}
+  </button>
+</div>
+<div class="cvi-tabs__list" role="tablist" *ngIf="currentWindowWidth < 992">
+  <button
+    class="cvi-tabs__mobile-button"
+    type="button"
+    id="dropdownMenuButton"
```

Comment:
All `id`s in a component must be unique – otherwise they will clash if a component is used multiple times on a page.